nms/app/components/context/CbsdContext.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright 2020 The Magma Authors.
```

Comment:
2022?

Reply:
At moment all headers are for 2020 and there is a check with a hard coded year.

Reply:
Thanks to https://github.com/magma/magma/pull/12714 this is now possible to hold an up to date date. :wink:

Reply:
Changes Copyright to 2022 in all files touched :)

Reply:
@maxhbr correct me if I'm wrong, but shouldn't only new files get the 2022 header?